torchtune/models/llama4/_model_builders.py

```diff
@@ -22,7 +28,7 @@
 
 def llama4_scout_17b_16e(
     decoder_trainable: bool = True,
-    vision_encoder_trainable: bool = False,
+    encoder_trainable: bool = False,
     fusion_trainable: bool = True,
```

Comment:
nit: possibly use all kwargs OR at the very least kwargs for image_size

Reply:
Done